test/performance/latency_test.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 func TestTimeToServeLatency(t *testing.T) {
-	perfClients, err := Setup(t.Logf, true)
+	perfClients, err := Setup(t.Logf, true, false)
```

Comment:
Signatures with multiple boolean arguments are nearly impossible to read.

One thing you can do in Go is define an enumeration to replace this:

```
type Direction int

const (
    North Direction = iota
    East
    South
    West
)
```

(source: https://yourbasic.org/golang/iota/)

This makes things much more readable:

```suggestion
	perfClients, err := Setup(t.Logf, EnableBacon, DisableSyrup)
```

apparently I'm hungry :)

Reply:
Updated. Thanks for pointing it out. I really like this approach :)